install.sh

```diff
@@ -4,23 +4,25 @@
 
 set -e
 
-case $(uname -s) in
+case "$(uname -s)" in
 Darwin) target="x86_64-apple-darwin" ;;
 *) target="x86_64-unknown-linux-gnu" ;;
 esac
 
-if [ $(uname -m) != "x86_64" ]; then
+if [ "$(uname -m)" != "x86_64" ]; then
```

Comment:
This change is fine. :) It is already adressed by https://github.com/denoland/deno_install/pull/121, however.